cluster/src/dunit/scala/io/snappydata/externalstore/ColumnTableDUnitTest.scala

```diff
@@ -631,6 +631,19 @@ class ColumnTableDUnitTest(s: String) extends ClusterManagerTestBase(s) {
     assert(snc.sql("select count(*) from airline").count()>0)
     snc.sql("drop table airline")
   }
+
+  def testSNAP1210(): Unit = {
+    val snc = org.apache.spark.sql.SnappyContext(sc)
+
+    snc.sql(s"create table t1 using com.databricks.spark.csv options(path " +
+      s"'${(getClass.getResource("/northwind/orders.csv").getPath)}', header 'true', " +
+      s"inferschema 'true')")
+    snc.sql("select * from t1").printSchema()
+    snc.sql("select * from t1").show
+    snc.sql("select * from t1").write.csv("/tmp/" + System.currentTimeMillis())
+    assert(snc.sql("select count(*) from t1").count() > 0)
```

Comment:
you may want to remove the csv file at the end of test else files will keep building in /tmp